src/Propel/Generator/Behavior/Validate/ValidateBehavior.php

```diff
@@ -216,7 +205,11 @@ protected function addLoadValidatorMetadataMethod()
                     $classConstraint = "Propel\\Runtime\\Validator\\Constraints\\".$properties['validator'];
                 }
             } else {
-                $classConstraint = "Symfony\\Component\\Validator\\Constraints\\".$properties['validator'];
+                if ($properties['validator'] === 'Unique') {
```

Comment:
SF introduced a new Unique validator after Propel stopped receiving updates, this caused issues with which loader is preferred.

SF's unique is not a suitable replacement for Propel's.